src/lib/libwebaudio.js

```diff
@@ -58,11 +58,10 @@ var LibraryWebAudio = {
 #if ASSERTIONS
     if (!ctx) console.error('emscripten_create_audio_context failed! Web Audio is not supported.');
 #endif
-    options >>= 2;
 
     var opts = options ? {
-      latencyHint: HEAPU32[options] ? UTF8ToString(HEAPU32[options]) : undefined,
-      sampleRate: HEAP32[options+1] || undefined
+      latencyHint: UTF8ToString({{{ makeGetValue('options', C_STRUCTS.EmscriptenWebAudioCreateAttributes.latencyHint, '*') }}}) || undefined,
```

Comment:
Is the logic here the same?   Won't UTF8ToString return an empty string if the ptr is null?